cmd/coverage.html

```diff
@@ -0,0 +1,309 @@
+
```

Comment:
I would add this file to the gitignore, too.

Reply:
Got it. Add this to next .gitignore for next branch.